usgs-details.html

```diff
@@ -182,35 +180,55 @@
         }
 
         // Create header text for live and fallback values
-        function renderHeader(metric, isFallback) {
-            var firstWord = isFallback ? 'Typical ' : 'Current ';
-            return firstWord + metric.display + ' in the Schuylkill River:';
+        function renderHeader(metric, isFallback, isBackupSensor) {
```

Comment:
Hm. I can see the logic in this but I think I'm inclined to leave these arguments lists as is for now. Aside from string constants and the metrics object, I don't think we've set any other global variables. If we end up doing #23, we should consider creating some app state there.